apps/webservice/src/app/[workspaceSlug]/(targets)/target-providers/integrations/google/UpdateGoogleProviderDialog.tsx

```diff
@@ -32,8 +36,9 @@ export const UpdateGoogleProviderDialog: React.FC<{
   providerId: string;
   name: string;
   projectIds: string[];
+  parentCloseControl?: () => void;
   children: React.ReactNode;
-}> = ({ providerId, name, projectIds, children }) => {
+}> = ({ providerId, name, projectIds, parentCloseControl, children }) => {
```

Comment:
Can we handle the open close state at the parent level? this onClose stuff feels weird